app/views/handlers/question.py

```diff
@@ -79,6 +79,10 @@ def rendered_block(self):
         )
 
         self._set_page_title(page_title)
+
+        if transformed_block["type"] == "ListCollectorContent":
+            return transformed_block
```

Comment:
Since this is `question.py`, just I suggest checking if `question` exists, if not return the block. No need to couple it to the type.